bindings/python/examples/wallet/get_client.py

```diff
@@ -0,0 +1,10 @@
+from iota_sdk import Wallet
```

Comment:
are we sure this example is needed? :D we already have so many :/

Reply:
Many examples are good to have, we just need to structure them better but that's in progress already
If we have another example that shows this being used, for example the faucet one in https://github.com/iotaledger/iota-sdk/pull/397, I'm fine if we remove this again, but I would like to have at least one where this is shown